dbms/src/Flash/Coprocessor/JoinInterpreterHelper.cpp

```diff
@@ -453,5 +450,20 @@ std::vector<RuntimeFilterPtr> TiFlashJoin::genRuntimeFilterList(
     }
     return result;
 }
+
+NamesAndTypes genDAGExpressionAnalyzerSourceColumns(Block block, const NamesAndTypes & tidb_schema)
```

Comment:
The logic of this function makes me a little confused...
Can you add some comments?

Reply:
OK, added.